config/manager/manager.yaml

```diff
@@ -66,6 +67,9 @@ spec:
           requests:
             cpu: 20m
             memory: 50Mi
+          limits:
+            cpu: 100m
+            memory: 250Mi
```

Comment:
Seems like the limits here and in the bundle CSV don't match. Do we need another `make bundle` run (after setting the limit to 500Mi)?

Reply:
Oh yeah, maybe. I'll change this to 500 and see if make bundle changes it. Thanks!

Reply:
Yup, just forgot :)